app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,14 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".UIActivity" android:screenOrientation="landscape"
+            >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
```

Comment:
Do we want 2 launcher main activities? Is the idea to have one for portrait and one for landscape?